orga/app/components/charts/participants-by-day.js

```diff
@@ -0,0 +1,167 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import moment from 'moment';
+import { TOOLTIP_CONFIG } from './chart';
+
+export default class ParticipantsByDay extends Component {
+  @service store;
+  @service intl;
+  @tracked days = 0;
+  @tracked startedDatasets = [];
+  @tracked sharedDatasets = [];
+
+  @tracked loading = true;
+
+  constructor(...args) {
+    super(...args);
+    const { campaignId, isTypeAssessment } = this.args;
+    this.isTypeAssessment = isTypeAssessment;
```

Comment:
[question]
Pourquoi ne pas utiliser directement `this.args.isTypeAssessment` où c'est nécessaire au lieu de créer un nouvelle variable `isTypeAssessment`

Reply:
done